libbeat/otelbeat/beatconverter/beatconverter.go

```diff
@@ -48,19 +48,20 @@ func (c converter) Convert(_ context.Context, conf *confmap.Conf) error {
 	for _, beatreceiver := range supportedReceivers {
 		var out map[string]any
 
+		var accessString = "receivers::" + beatreceiver
```

Comment:
Can we rename it to something like `beatReceiverConfigKey`?
`accessString` does not really describe it well enough.